lib/shopify_api/resources/usage_charge.rb

```diff
@@ -0,0 +1,13 @@
+module ShopifyAPI
+  class UsageCharge < Base
+    init_prefix :recurring_application_charge
+
+    class << self
+      def index(recurring_application_charge_id)
+        UsageCharge.find(:all, :params => { :recurring_application_charge_id => recurring_application_charge_id })
+      rescue ActiveResource::ForbiddenAccess
```

Comment:
Sorry - this is the client API already - this *is* the 403 coming back from the server